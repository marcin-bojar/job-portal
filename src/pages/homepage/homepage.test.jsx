import React from 'react';
import { shallow } from 'enzyme';

import HomePage from './homepage.component';

it('renders HomePage', () => {
  expect(shallow(<HomePage />)).toMatchSnapshot();
});
