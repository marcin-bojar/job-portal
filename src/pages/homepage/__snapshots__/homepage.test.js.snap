// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders HomePage 1`] = `
<div
  className="homepage"
>
  <div
    className="hero-header"
  >
    <h1>
      <span
        className="hero-header__color-letter"
      >
        C
      </span>
      argo
       
      <span
        className="hero-header__color-letter"
      >
        R
      </span>
      unner
    </h1>
    <h2>
      Praca w logistyce
    </h2>
  </div>
  <div
    className="data-container"
  >
    <div
      className="data-container__ads"
    >
      <AdsPreview />
    </div>
    <div
      className="data-container__search"
    />
  </div>
</div>
`;
